platform/linux-generic/include/odp_config_internal.h

```diff
@@ -141,7 +141,7 @@ extern "C" {
  * all ODP threads (when the _ODP_ISHM_SINGLE_VA flag is used).
  * In bytes.
  */
-#define ODP_CONFIG_ISHM_VA_PREALLOC_SZ (536870912L)
+#define ODP_CONFIG_ISHM_VA_PREALLOC_SZ (1073741824L)
```

Comment:
Would it be better to change this to `#define ODP_CONFIG_ISHM_VA_PREALLOC_SZ (1024 * 1024 * 1024L)` to make it clearer what this number is?

Reply:
Makes sense, will update.